sig-scalability/annual-report-2022.md

```diff
@@ -24,58 +24,59 @@
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Each subproject could benefit from additional hands. However, the Scalability Test Frameworks and Scalability and
+     Performance tests and validation are the ones where we can grow contributors - the other require both very deep and
+     wide understanding of Kubernetes before making reasonable contributions.
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - We care mostly about metrics for our main repository [perf-tests]. We measure them through [devstats]. We keep track of number of reviewers, opened issues and PRs merged.
 
 3. Does your [CONTRIBUTING.md] help **new** contributors engage with your group specifically by pointing
    to activities or programs that provide useful context or allow easy participation?
 
-   -
+   - Yes, [CONTRIBUTING.md] points to maintained list of issues with label `good first issue`.
 
 4. If your group has special training, requirements for reviewers/approvers, or processes beyond the general [contributor guide],
    does your [CONTRIBUTING.md] document those to help **existing** contributors grow throughout the [contributor ladder]?
 
-   -
+   - We don't have any special requirements for reviewers/approvers, but most of core areas of sig-scalability required
+     very deep and wide understanding of the whole Kubernetes. It is much easier to start with our testing infrastructure.
 
 5. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, we have contributors from [multiple companies](https://k8s.devstats.cncf.io/d/8/company-statistics-by-repository-group?orgId=1&var-period=d7&var-metric=contributions&var-repogroup_name=SIG%20Scalability&var-repo_name=kubernetes%2Fkubernetes&var-companies=All&from=1641013200000&to=1672462800000).
 
 6. Are there ways end users/companies can contribute that they currently are not?
    If one of those ways is more full time support, what would they work on and why?
 
-   -
-   -
+   - End users usually come with questions about scalability, they rarely want to contribute.
```

Comment:
Maybe one way end users could help provide inputs to this SIG are let us know of ways in which they want to scale their kubernetes clusters? I believe this kind of happens time to time today (slack/zoom calls), but we could have them record that somewhere. Wdyt?

Reply:
added